CMSgov/mint-app
# [EASI-3934] Dependency Updates  -- Round 1

# EASI-3934


## Changes and Description

- Updated the following to the latest version as of 2/16:
  - `@apollo/client`
  - `@babel/core`
  - `@babel/preset-env`
  - `@types/react-table`
  - `classnames`
  - `cypress`
  - `detect-browser`
  - `json2csv`
  - `okta/okta-react`
  - `okta/okta-sigin-widget`
  - `react-ga4`
  - `react-modal`
  - `react-paginate`
  - `react-select`
  - `react-to-print`
  - `redux-devtools-extension`
  - `sass`
  - `storybook`
  - `wait-on`
  - `yup`
- Removed the following:
  - `@types/yup` since the types are bundled in `yup`(<https://www.npmjs.com/package/@types/yup>)
  - `js-base64` since it is not being used anywhere in the codebase
- Ran `npx browserslist@latest --update-db` to update the browserlist 

## How to test this change

1. `yarn install`
2. Bring up app
3. Navigate through app to see if anything breaks
4. Try out the CSV
5. Try out storybook



## PR Author Review Checklist


## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
